curriculum/challenges/english/25-front-end-development/workshop-musical-instrument-inventory/6895e356fd877efd6e3657ee.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 6895e356fd877efd6e3657ee
+title: Step 8
+challengeType: 20
+dashedName: step-8
+---
+
+# --description--
+
+Methods are functions defined inside a class that can perform actions using the class's data. 
+
+Here is how you create methods:
+
+```python
+def method_name(parameter1, parameter2,...):
+    # Method contents will go here
+```
```

Comment:
This is exactly what has been done in step 2. I don't think we need to repeat the same instructions.
```suggestion
```

Reply:
I forgot to remove these.